net/sixlowpan/sixlowpan_tcpsend.c

```diff
@@ -242,33 +242,19 @@ static int sixlowpan_tcp_header(FAR struct tcp_conn_s *conn,
   memcpy(ipv6tcp->tcp.ackno, conn->rcvseq, 4);    /* ACK number */
   memcpy(ipv6tcp->tcp.seqno, conn->sndseq, 4);    /* Sequence number */
 
-  /* Set the TCP window */
+  /* Update the TCP received window based on I/O buffer availability */
 
-  if (conn->tcpstateflags & TCP_STOPPED)
```

Comment:
@xiaoxiang781216 The flag is checked right here. It sets the window to zero so that the host stops sending data. Why do you say that the code is dead?

Reply:
Sorry, I mean there isn't code to set STOPPED flag, not no check:(.

Reply:
I see.  I can't seem to find whether it was ever set or not.  Alot has changed.  I can try and dig into it later, but not sure when I'll have time. If it wasn't ever fully supported, then it likely came from the Contiki port that Greg originally did. 

In either case - why get rid of this logic?  It seems like the right behavior, even if right now we don't look for the STOPPED flag right now.  If we were to look for the STOPPED flag, then this would be right.  What's the rationale for removing it? 

The STOPPED flag is a legitimate TCP flag.  This logic, likely copied from Contiki, is correct.  I think we should either set the STOPPED flag and allow this logic to do it's job, or not remove it and leave it as is.

Reply:
The problem is that the code doesn't work as expect. If someone plan to implement shutdown function in the future, he will bring back the similar code.

Reply:
Can you elaborate on what doesn't work as expected? Maybe we can just fix it?

Reply:
Because nobody set STOPPED flag, one improvement is implement shutdown as @patacongo mention which will set STOPPED definitely.